Library/Formula/hidapi.rb

```diff
@@ -2,8 +2,8 @@
 
 class Hidapi < Formula
   homepage 'https://github.com/signal11/hidapi'
-  url 'https://github.com/signal11/hidapi/archive/hidapi-0.8.0-rc1.tar.gz'
-  sha1 '5e72a4c7add8b85c8abcdd360ab8b1e1421da468'
+  url 'https://github.com/rbarazzutti/hidapi/archive/hidapi-0.8.0-rc1-with-macfix.tar.gz'
+  sha1 '8fa3f991839713be48279e6047b74df7f4e2c4df'
```

Comment:
Rather than fork can you please submit this patch to the upstream developers of this project as a pull request and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!